scripts/ci/run-test-default.sh

```diff
@@ -58,6 +58,7 @@ ${TESTCMD} --label=System.Configuration --timeout=5m make -w -C mcs/class/System
 ${TESTCMD} --label=System.Transactions --timeout=5m make -w -C mcs/class/System.Transactions run-test
 ${TESTCMD} --label=System.Web.Extensions --timeout=5m make -w -C mcs/class/System.Web.Extensions run-test
 ${TESTCMD} --label=System.Core --timeout=15m make -w -C mcs/class/System.Core run-test
+${TESTCMD} --label=System --timeout=5m make -w -C mcs/class/System run-xunit-test
```

Comment:
change it to --label=System-xunit please. also move it up to the System run, it's below System.Core right now :)